ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -374,6 +386,8 @@ literals:
   - OFF
   - AUTO
   - MANUAL
+  - DRY
```

Comment:
dry? what does this mean in the context of a running stand-alone fan or pump (which is where this field is allowably used)?

Reply:
Used in FCU not standalone fan